plans/install.pp

```diff
@@ -115,16 +155,18 @@
       | HEREDOC
   )
 
-  run_task('pe_xl::mkdir_p_file', $puppetdb_database_replica_host,
-    path    => '/etc/puppetlabs/puppet/csr_attributes.yaml',
-    content => @("HEREDOC"),
+  if $ha {
+    run_task('pe_xl::mkdir_p_file', $puppetdb_database_replica_host,
+      path    => '/etc/puppetlabs/puppet/csr_attributes.yaml',
+      content => @("HEREDOC"),
       ---
```

Comment:
I thin lines 162-167 should be indented one more level.

Reply:
Updated